tests/support_server/configure.pm

```diff
@@ -49,6 +57,43 @@ sub run {
 
     script_output($script);
 }
+sub _firewall {
+    assert_script_run 'sysctl net.ipv4.ip_forward';
+}
+sub _packages {
```

Comment:
Could you please make your function names to be containing verbs as currently it is not clear what they exactly do.